sqlmesh/core/plan/builder.py

```diff
@@ -801,27 +800,6 @@ def _ensure_valid_date_range(self) -> None:
                     f"Plan end date: '{time_like_to_str(end)}' cannot be in the future (execution time: '{time_like_to_str(self.execution_time)}')"
                 )
 
-    def _ensure_no_forward_only_revert(self) -> None:
```

Comment:
Why does allowing users to not have a VDE prod mean that we no longer need this check?